superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -53,6 +54,84 @@ const createFetchResourceMethod = (method: string) => (
   return [];
 };
 
+export const getBatchData = (userId: string | number, recent: string) => {
```

Comment:
Does this get recent objects? If so, how about `getRecentActivityObjects`? How are we thinking about "recently viewed?"

Reply:
getRecentActivityObjects makes sense!